DiamondLightSource/SynchWeb
# [LIMS-145]Feature: Change request dispatched email recipient

**JIRA ticket:** [LIMS-145](https://jira.diamond.ac.uk/projects/LIMS/issues/LIMS-145)

**Changes:**

- Make 'Country' a separate dropdown in the dispatch request form
- Add country to dispatch model
- If country is not the same as `$facility_country`, send email to new global variable `$dispatch_email_intl` rather than `$dispatch_email`.

**To test:**

- Check that the dispatch email is sent to the correct recipients for countries the same as, and different to the facility country
- Check that dewar ispatch is not broken in the case where `$dispatch_email_intl` isn't defined in `config.php`

**Note:** Feature requires the global variable `$dispatch_email_intl` to be defined in `config.php`
